src/oge/emissions.py

```diff
@@ -705,6 +726,7 @@ def calculate_generator_nox_ef_per_unit_from_boiler_type(gen_fuel_allocated, yea
         gen_nox_factors.loc[
             gen_nox_factors["emission_factor"].isna(),
             [
+                "plant_id_eia",
```

Comment:
See above: For this check, we do not want to add plant IDs or BA codes - we only care about generic prime movers across all plants.